maia-core/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+pub use secp256k1_zkp;
+
+pub mod interval;
+
+mod protocol;
+
+pub use protocol::{
+    generate_payouts, Announcement, Cets, CfdTransactions, PartyParams, Payout, PunishParams,
+    TransactionExt, TxBuilderExt, DUMMY_2OF2_MULTISIG,
```

Comment:
🤔 This just reminded me that it makes no sense that this library export the `TransactionExt` trait. We should really duplicate this in `itchysats` and keep it private here.

Reply:
do you mean moving it? how come duplicating would be a cleaner solution?